doc/source/examples/extended_examples/hpc/hpc_ml_ga.rst

```diff
@@ -0,0 +1,447 @@
+        mapdl.solve()
+
+        # Extracting data
+        UZ = mapdl.post_processing.nodal_displacement("Z")
+        UZ_node_12 = UZ[12]
+
+        return UZ_node_12
+
+As it can be seen, this function returns the control parameter for this model, which is the displacement at the Z-direction on the node 12 (``UZ_node_12``).
+
+
+Setting MAPDL pool
+==================
+
+The genetic algorithms are expensive regarding the amount of calculation needed to reach an optimal solution. As seen before, many simulations must be performed to select, crossover, and mutate
+all the chromosomes across all the populations.
+For this reason, to speed up the process, it is desirable to have as many MAPDL instances as possible, so
+each one can calculate one chromosome fit function.
+
+To manage multiple MAPDL instances, the best approach is to use :py:class:`~ansys.mapdl.core.pool.MapdlPool` class.
+
+.. code-block:: python
+
+    from ansys.mapdl.core import MapdlPool
+
+    # Starting pool
+    # Number of instances should be equal to number of CPUs
+    # as set later in the ``sbatch`` command
+    pool = MapdlPool(n_instances=10)
+    print(pool)
+
+
+Define deflection target
+========================
+
+Because this is a demonstration example, the target displacement is calculated 
+using the beam function itself using a force of 22840 :math:`N/cm^2`.
+
+.. code-block:: python
+
+    # Calculate target displacement
+    mapdl = pool[0]
+    force = 22840  # N/cm2
+    target_displacement = calculate_beam(pool[0], [force])
+    print(f"Setting target to {target_displacement} for force {force}")
+
+
+Genetic algorithm model
+=======================
+
+Introduction
+------------
+
+The library `PyGAD <pygad_docs_>`_ is used to configure the genetic algorithm.::
+
+    PyGAD is an open-source Python library for building the genetic algorithm and optimizing machine learning algorithms.
+
+    PyGAD supports different types of crossover, mutation, and parent selection operators.
+    PyGAD allows different types of problems to be optimized using the genetic algorithm
+    by customizing the fitness function. It works with both single-objective and
+    multi-objective optimization problems.
+
+
+Configuration
+-------------
+
+To configure the genetic algorithm, the following code is used:
+
+.. code-block:: python
+
+    # Setting GA model
+    sol_per_pop = 20
+    num_generations = 10
+    num_parents_mating = 20
+    num_genes = 1  # equal to the size of inputs/outputs.
+    parallel_processing = ["thread", len(pool)]  # Number of parallel workers
+
+    # Initial guess limits
+    init_range_low = 10000
+    init_range_high = 30000
+    gene_type = int  # limit to ints
+
+    # Extra configuration
+    # https://blog.derlin.ch/genetic-algorithms-with-pygad
+    parent_selection_type = "rws"
+    keep_parents = 0  # No keeping parents.
+    mutation_percent_genes = 30
+    mutation_probability = 0.5
+
+
+From the above code, the most import parameters are:
```

Comment:
```suggestion
In the preceding code, the most import parameters are:
```
Style guide says not to use positional indicators like above and below (primarily because in a PDF they might be incorrect).